src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_RENTAL_COST = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM_DURATION = 7;
+  const SHORT_TERM_DURATION = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const TOTAL_PRICE = DAY_RENTAL_COST * days;
+
+  if (days >= LONG_TERM_DURATION) {
+    return TOTAL_PRICE - LONG_TERM_DISCOUNT;
```

Comment:
The discount is applied directly to the `TOTAL_PRICE`, which is calculated without considering any discounts. Ensure that the discount is subtracted from the correct total price calculation.